src/frontend/next/src/components/Posts/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState, useEffect } from 'react';
```

Comment:
This file should be named Posts.tsx

Reply:
Okay. So keep it that way. I will fix something on the main page, and then I will relaunch CI/CD so we can see it on the netlify deployment.